code/datums/effects/tether.dm

```diff
@@ -4,7 +4,7 @@
 	var/range = 0
 	var/datum/effects/tethered/tethered
 	var/tether_icon // The icon used for the Beam proc for the tether
-	var/beam_id
+	var/datum/beam/beam_id
```

Comment:
A better name could be used here given it's no longer an ID but the actual beam.